modules/itc/src/main/scala/lucuma/itc/search/syntax/Wavelength.scala

```diff
@@ -15,16 +17,11 @@ final class WavelengthOps(val self: Wavelength) extends AnyVal {
 
   /** Returns the difference of this wavelength and `other`, clipped at Wavelength.Min. */
   def -(other: Wavelength): Wavelength =
-    Wavelength.fromPicometers
-      .getOption(self.toPicometers.value.value - other.toPicometers.value.value)
-      .getOrElse(Wavelength.Min)
+    Wavelength(self.toPicometers - other.toPicometers).max(Wavelength.Min)
```

Comment:
In retrospect, I'm not sure if this will do the right thing. The `AdditiveGroup[PosInt]` will never return a negative.